cmd/mothership/mothership/start/helpers.go

```diff
@@ -18,3 +21,36 @@ func components(cfg model.ClusterConfigurationEntity) []keb.Component {
 	}
 	return components
 }
+
+func filterReconciliationsAfter(time time.Time, reconciliations []keb.Reconciliation) []keb.Reconciliation {
```

Comment:
Would move this code into a filter (see https://github.com/kyma-incubator/reconciler/blob/main/pkg/scheduler/reconciliation/filter.go)

The `reconciliation.Repositroy` should filter that either by defining a SQL query using a WHERE condition (in `Filter.FilterByQuery` or by checking the creation-time in `Filter.FilterByInstance` << this can use the code you have implemented in your helper.go)